src/sdkRuntimeModels.ts

```diff
@@ -163,6 +163,7 @@ export interface SDKProduct {
 
 export interface MParticleWebSDK {
     addForwarder(mockForwarder: MPForwarder): void;
+    _instanceName: string;
```

Comment:
This will be part of #968

Reply:
Can you clarify wha tthis means?  I don't see it in that PR

Reply:
Oh, I think I get it.  Basically, it will also be included in the TS migration but under IMParticleWebSDKInstance, right?  Just didn't see it yet.